hazelcast-wm/src/test/java/com/hazelcast/wm/test/WebFilterClientFailOverTests.java

```diff
@@ -0,0 +1,97 @@
+package com.hazelcast.wm.test;
+
+import com.hazelcast.config.FileSystemXmlConfig;
+import com.hazelcast.core.Hazelcast;
+import com.hazelcast.core.IMap;
+import org.apache.http.client.CookieStore;
+import org.apache.http.impl.client.BasicCookieStore;
+import org.junit.Test;
+
+import java.io.File;
+
+import static org.junit.Assert.assertEquals;
+
+public abstract class WebFilterClientFailOverTests extends AbstractWebFilterTest {
+
+    String testName;
+    protected WebFilterClientFailOverTests(String serverXml1) {
+        super(serverXml1);
+    }
+
+    protected WebFilterClientFailOverTests(String serverXml1, String serverXml2) {
+        super(serverXml1, serverXml2);
+        testName = serverXml1;
+    }
+
+    @Test
+    public void whenClusterIsDown_enabledDeferredWrite() throws Exception {
+        CookieStore cookieStore = new BasicCookieStore();
+        assertEquals("true", executeRequest("write", serverPort1, cookieStore));
+        assertEquals("value", executeRequest("read", serverPort1, cookieStore));
+        hz.shutdown();
```

Comment:
please add a whiteline after before the shutdown and the other steps. Then the layout explains you have multiple steps.